app/views/users/index.html.erb

```diff
@@ -7,5 +7,14 @@
 
   <body>
     <h1> This is the Users page</h1>
+    <% @users.each do |user| %>
+      <a href="/users/<%= user.id%>">
+        <div></div>
+        <div>
+          <h2><%= user.name %></h2> 
+          <span>Number Of Posts: <%= user.posts_counter %></span>
+        </div>
+      </a>
+    <% end %>
```

Comment:
- [ ] Please make sure that the user index page matches the design as you can see the title of the page is not present in the design also you have to add boxes that contain the username and the number of posts for that user and this box should be horizontal as in the design and you have to add a place holder to the users profile images.

| Yours  | design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/216770503-03953011-f825-423a-8d55-4e2601fee114.png) | ![image](https://user-images.githubusercontent.com/74744787/216770520-ea37a7d7-6fe9-40f1-88c5-3f75a2293ede.png)  |